src/sqlancer/common/DBMSCommon.java

```diff
@@ -0,0 +1,30 @@
+package sqlancer.common;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public final class DBMSCommon {
+
+    private static final Pattern sqlancerIndexPattern = Pattern.compile("^i\\d+");
```

Comment:
I think this should be `SQLANCER_INDEX_PATTERN` according to the Java naming convention.

Reply:
Ahh yes thank you!